### ISSUE TITLE
Inconsistency of AMD supports (eve & raphael)

### ISSUE THREAD

**Issue by FrankFang** [DmitryBaranovskiy/raphael#948]
~~~js
//eve.js
(typeof define === "function" && define.amd ? (define("eve", [], function() { return eve; })) : (glob.eve = eve));
~~~

~~~js
// raphael.js
if (typeof define === "function" && define.amd) {
        // Define as an anonymous module
        define(["eve"], function( eve ) {
            return factory(glob, eve);
        });
    }
~~~

Why not make them same?

**Comment by tomasAlabes** [comment]
Already done like that.
Thanks.

**Comment by vincentwinkel** [comment]
Hi,
I reopen this issue because I've a problem with Magento2 requirejs config. (I precise I'm not expert on requirejs).

I correctly included Raphael 2.2.0 as a dependency of my module, like that :
``` js
define([
    'raphael', //With a mapping on my file, correctly HTTP downloaded
], function (Raphael) {
    //Raphael is undefined
}
```
After checking the source, I see line 14 : `define("Raphael", [], factory);`
And according to the [requirejs doc](http://requirejs.org/docs/api.html#modulename), it seems not good to manually define a module name.
So I tried to remove the first arg "Raphael" , and it works !

Is it possible to fix this issue ? Or if it's not a bug but a misunderstanding from me, I apologize.
Thanks

**Comment by tomasAlabes** [comment]
Done. v2.2.1
